### PR TITLE
Modified abstract ops multinode

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Abstract_ops_multinode needs to check the error code for each ret
returned. If it fails in any case the exception has to be raised.

Fixes: #258

Signed-off-by: Ayush Ujjwal <aujjwal@redhat.com>

### CODE REVIEW

**Review by srijan-sivakumar — APPROVED** [review]
LGTM

File: common/ops/abstract_ops.py [hunk 1]
```diff
@@ -38,24 +38,25 @@ def execute_abstract_op_node(self, cmd : str, node : str=None):
...
 
         ret = self.execute_command_multinode(cmd, node)
 
-        if int(ret['msg']['opRet']) != 0:
-            self.logger.error(ret['msg']['opErrstr'])
-            raise Exception(ret['msg']['opErrstr'])
+        for each_ret in ret:
+            if each_ret['error_code'] != 0:
+                self.logger.error(each_ret['msg']['opErrstr'])
+                raise Exception(each_ret['msg']['opErrstr'])
```
Comment (nishith-vihar) [inline common/ops/abstract_ops.py:59]: This part should follow the pattern in the execute_abstract_op_node

**Review by nishith-vihar — CHANGES_REQUESTED** [review]

File: common/ops/abstract_ops.py (hunk 1 above)
Comment (aujjwal-redhat) [reply common/ops/abstract_ops.py:59]: Ack.

File: common/ops/abstract_ops.py [hunk 2]
```diff
@@ -38,24 +38,29 @@ def execute_abstract_op_node(self, cmd : str, node : str=None):
...
+            elif isinstance(each_ret['msg'],OrderedDict) or isinstance(each_ret['msg'],dict):
+            if int(each_ret['msg']['opRet']) != 0:
+                self.logger.error(each_ret['msg']['opErrstr'])
+                raise Exception(each_ret['msg']['opErrstr'])
```
Comment (nishith-vihar) [inline common/ops/abstract_ops.py:63]: Indentation

Comment (aujjwal-redhat) [reply common/ops/abstract_ops.py:63]: Oh yeah. My bad :)

**Review by nishith-vihar — APPROVED** [review]

**Pull request merged**